GarbageCan.Application/Boosters/ActiveBoosters/Queries/GetActiveBoostersQuery.cs

```diff
@@ -7,7 +7,6 @@ namespace GarbageCan.Application.Boosters.ActiveBoosters.Queries
 {
     public class GetActiveBoostersQuery : IRequest<ActiveBooster[]>
```

Comment:
should this be renamed to `GetAllActiveBoostersQuery`?

Reply:
Renamed!